bokehjs/src/lib/models/annotations/title.ts

```diff
@@ -69,24 +86,34 @@ export class TitleView extends TextAnnotationView {
 
   protected _render(): void {
     const {text} = this.model
-    if (text == null || text.length == 0)
+    if (text == null)
       return
 
     this.model.text_baseline = this.model.vertical_align
     this.model.text_align = this.model.align
 
     const [sx, sy] = this._get_location()
-    const angle = this.panel.get_label_angle_heuristic("parallel")
+    const position = {
+      sx,
+      sy,
+      x_anchor: this.model.align,
+      //y_anchor: this.model.vertical_align,
```

Comment:
It is counterintuitive, but this vertical_align is not the anchor. The anchor is where to start the drawing from, images always start on top left and text we usually give it the alphabetical baseline. So here you need to set the correct Y anchor and MathTextView will find where to draw